package.json

```diff
@@ -1,50 +1,45 @@
 {
   "name": "react-page-visibility",
-  "version": "0.0.2",
+  "version": "1.0.0",
   "description": "Declarative, nested, stateful, isomorphic page visibility for React",
   "author": "Gilad Peleg <giladp007@gmail.com> (http://giladpeleg.com)",
   "repository": "pgilad/react-page-visibility",
-  "keywords": "react, components, visibility, page, higher-order, browser",
-  "main": "index.js",
+  "keywords": "react, components, visibility, page, higher-order, browser, react-component",
+  "main": "dist/index.js",
+  "module": "src/index.js",
```

Comment:
Unfortunately, this will still cause the same issue that I raised in #1. I've tested this locally. You are using ES6 features which haven't been transpiled to ES5, therefore it will cause uglify-js to error on a production build in a project which consumes this package. 

I've seen that most people put the transpiled code into a `lib` folder, I believe it's also a bad practice to point straight to your `src` directory.